### PR TITLE
Accession spawn Resource

### PR DESCRIPTION
**Pull request by kdimopulu**

### CODE REVIEW

File: staff_features/accessions/accession_spawn_resource.feature [hunk 1]
```diff
@@ -2,19 +2,20 @@ Feature: Resource Spawn from an Accession
...
-      | A new Resource has been spawned from Accession "Original Accession Title". This record is unsaved. You must click Save for the record to be created in the system. |
+     And the Resource has been spawned from Accession info message is displayed
   Scenario: Original Accession record is linked to the spawned Resource
-    Given the user is on the New Resource page spawned from the original Accession
-     When the user click on 'Related Accessions' at the sidebar
-     Then the 'Related Accessions' form is displayed
-      And the original Accession Title is filled in
+     When the user clicks on 'Spawn'
+      And the user clicks on 'Resource' in the dropdown menu
+     Then the New Resource page is displayed
```
Comment (kdimopulu) [inline staff_features/accessions/accession_spawn_resource.feature:14]: And the Resource has been spawned from Accession info message is displayed

File: staff_features/accessions/accession_spawn_resource.feature [hunk 2]
```diff
@@ -2,19 +2,20 @@ Feature: Resource Spawn from an Accession
...
-     And the following info message is displayed
-      | A new Resource has been spawned from Accession "Original Accession Title". This record is unsaved. You must click Save for the record to be created in the system. |
+     And the Resource has been spawned from Accession info message is displayed
   Scenario: Original Accession record is linked to the spawned Resource
-    Given the user is on the New Resource page spawned from the original Accession
-     When the user click on 'Related Accessions' at the sidebar
-     Then the 'Related Accessions' form is displayed
-      And the original Accession Title is filled in
+     When the user clicks on 'Spawn'
+      And the user clicks on 'Resource' in the dropdown menu
```
Comment (kdimopulu) [inline staff_features/accessions/accession_spawn_resource.feature:13]: in the spawn dropdown menu

Comment (kdimopulu) [inline staff_features/accessions/accession_spawn_resource.feature:13]: in the spawn dropdown menu

File: staff_features/accessions/accession_spawn_resource.feature [hunk 3]
```diff
@@ -2,19 +2,20 @@ Feature: Resource Spawn from an Accession
   Background:
    Given an administrator user is logged in
      And an Accession has been created
-     And the Accession with the title "Original Accession Title" is opened in edit mode
+     And the Accession is opened in edit mode
   Scenario: Resource Spawn from Accession page is opened
```
Comment (kdimopulu) [inline staff_features/accessions/accession_spawn_resource.feature:6]: to be deleted

**Review by donaldjosephsmith — APPROVED** [review]

**Pull request merged**